src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -284,7 +291,7 @@ public String print() {
             }
             return json;
         } catch (JsonProcessingException | RepositoryException e) {
-            // ignore
+			log.error("Failed to generate JSON string", e);
```

Comment:
Yeah, this makes sense for JsonProcessingExceptions. However, a RepositoryException appears more serious to me, so we should maybe differentiate them.